cmd/fleet/serve.go

```diff
@@ -272,15 +274,13 @@ the way that the Fleet server works.
 
 			// Flush seen hosts every second
```

Comment:
Comment doesn't match anymore.

Also, the randomness is added in case multiple instances of fleet start all "at the same time"?

Reply:
> Also, the randomness is added in case multiple instances of fleet start all "at the same time"?

Just to make reduce the chances of two instances competing for updating the seen time in bulk.